qiskit_experiments/library/randomized_benchmarking/clifford_utils.py

```diff
@@ -163,6 +164,15 @@ def _synthesize_clifford_circuit(
         basis_gates = list(basis_gates)
     coupling_map = CouplingMap(coupling_tuple) if coupling_tuple else None
 
+    # special handling that does a basis change only
```

Comment:
Would the user want to pass `synthesis_method="basis_only"` or is it only intended to be used when the user passes `1Q_fixed`? For the latter case, I wonder if this `transpile` call would be better placed in the `clifford_1_qubit_circuit` call and the swapping of the `synthesis_method` could be skipped. Maybe that is too cluttered. I think at least a comment that this is only expected to be used as part of `basis_only` would be good.